.travis.yml

```diff
@@ -1,7 +1,17 @@
 language: python
+
 python:
   - "3.6"
+
+
 install:
   - pip install -r requirements.txt
+  - pip install coveralls codecov
+
 script:
-  - python3 -m pytest
+  - python3 -m unittest
```

Comment:
choose whether you want to run your tests solely using unittest or nosetest not both

Reply:
I've chosen unittest only in the latest commit